SHIR/setup.ps1

```diff
@@ -46,8 +54,15 @@ function RegisterNewNode {
         Start-Sleep -Seconds 15
     }
 
-    if (!$NODE_NAME) {
-        Start-Process $DmgcmdPath -Wait -ArgumentList "-RegisterNewNode", "$($AUTH_KEY)" -RedirectStandardOutput "C:\SHIR\register-out.txt" -RedirectStandardError "C:\SHIR\register-error.txt"
+    if ($ENABLE_AD -eq "true") {
+        Write-Log "Enable Expired Nodes Auto-Deletion"
+        if (!$AD_TIME) {
+            $AD_TIME = 600
+        }
+
+        Write-Log "Node Expiration Time In Seconds: $($AD_TIME)"
+        Start-Process $DmgcmdPath -Wait -ArgumentList "-RegisterNewNode", "$($AUTH_KEY)", "$($NODE_NAME)", "$($AD_TIME)" -RedirectStandardOutput "C:\SHIR\register-out.txt" -RedirectStandardError "C:\SHIR\register-error.txt"
```

Comment:
Is this documented somewhere?

Current container documantion(1) mentions about node leaks but does not mention usage of third argument for `-RegisterNewNode`


(1) https://learn.microsoft.com/en-us/azure/data-factory/how-to-run-self-hosted-integration-runtime-in-windows-container#limitations

Reply:
We will update the document soon.